lib/ain-evm/src/evm.rs

```diff
@@ -497,4 +497,37 @@ impl EVMServices {
             storage: vec![(storage_index, ain_contracts::u256_to_h256(new_balance))],
         })
     }
+
+    pub fn dst20_is_deployed(
```

Comment:
Conveys functionality more clearly.

```suggestion
    pub fn is_dst20_queued_or_deployed(
```